apps/supermariokart/super_mario_kart.star

```diff
@@ -0,0 +1,209 @@
+            """iVBORw0KGgoAAAANSUhEUgAAAEAAAAAgCAYAAACinX6EAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QA/wD/AP+gvaeTAAAAB3RJTUUH5gcHBDkKymZ67gAABwJJREFUaN7lmXtsFNcVxn93PLt+YOzFXu/srDGPPNokPGyHRxulqFLaJE3VqEkRUUFq0khFCW0tgyISVW1xEyWRk0AhNcQtNJVKqraIR6tIjSIKtEVAG1owG8Bp44YYvN7Zh42f2PuYvbd/rG1CgmF3wYGKT7ra0cy9Z+5+9zv3nHtGKJTimuE08MWR32sD7dr9+esDNzwB+rWewHiQgAUkrtCOEzAZf6WvSwIUcEzTeMrtJVHoxCHS97KBAJIKnMMJ1nWFqJUS8f9AgAJaEKy+rZppz77Kw+VQLBM5ETCoOflDN6xuqOeV1mPUoj5BgsglCkgklohgY6OjYyoPWk7byYVRQAJ+TWP1bdVUPb+R+6ZorH+mnnAohKZlZ19KieH1suqlV9ndI+n40fd55d9+qqW8cKYKpbJpEqmO5L+rFs6+X029s0YtnP0V9U7+URUUYZUilZUtRbtSTFcKlAIVALXI41OLtx9Uu0+eUvMXLFAj6s+pCVDzFyxQu0+eUou3H1SLPD4VGHnXaMvKBSSSY86TLL9zJUd/egC8SYKh93io4VtUBapo/uBlauOzERf1tssjCYTzdKZVmEzWJJGQlbX0L1xciIQsJmuSggqTcJ5O8mN9stJVWHSxovJpji45ANMTYCrk3DjWllYOv76P5XetpCX/BBKZ04TFSEOlxwsx/vQ0LY/KShOfT+dS3jFmQ8nz9nMhQCI5nR+gw9cBCQn78iACxAGPgpoERxsPsOLW1VgikhMBGRMlBDU1c9m1q4EdOyqorgaRm+gyjwJRcZYXp2+g+7FO2OpK6zV2FkwJ1UARUJzE0iPYws4+bmUITdOoqZnL5s3fZc6cZpSK0twsWLFC4feDzFJ8GSugQpXxwzOrMJpmUBouwpMqRrMc8KEGB4G/An8nrYgJhGn6aG5uoLp6C3AMIWxqa6GpSeD1Zm8vYwVoQlCap+Pqz+fbd9/E7TOreGHfEVqHoqgHe6FEInp0zISBPoHpha47cLvdhMMRXC4JCHp7we0GPYfXZqyAWGEfR2p/y1BhD5qAW8tKeH7RXcxKedB+U474eTm3b59Fc2cjpvRMGAFCgGUpVq0SRCIQicDKlYpgELJMFbIjoGC4lM+3PIr77M1s/Vcne95twywp5HsP3MOUiklMKs3ngZlzKHYWTJT7jxGQTIJlgW2n2+h1LsiYgJSSOAddPDNYx1BBPs2BNra9L9kw/DV6GhMMrYXXjYd4dPJawtrERQGlwOEA00xLXtfPX+eCzIblgWVG+GbZck6bp+n+SRsp5eGXW5Yx+CdQn02CUPTujzDkjBJ0WJhxI+eE6JILkZIYhmD9ekFZWfrehg2C/n5IpbK3l5kCDIi9McSZ352gc/0J7Glx1Ac2A3tAhYAzHjRrkNneX/PEV4v42Wdexu88iboKzqBp4PGM+reGaRoUFHThdicJhSAUSm+AhYVgGBcbcwUESCQhPUrQaXE2EUD5bTgEvJUH73tgAOj1wGvPUv7HO3hi/lS8niLuX1LOD6oaOKIfzzkrhHRM93ph0ybBvHkwb57Bpk2P4/E8RygUZOlSWLpUYVlpN2hqGu2XHuP1Xj4vGNcFJJLWSW2suWUdvVqQGU+2Mai7QCuDlAv6H4NzSRBhHMMO3CU3s7c/n2/c9zBDsRjHy16ifngN2yO/wGcbOZOgVHplt20TaFoXpvkcSkUJBFK0t6f7dHamCaithZ07BVJCYOTUczmMS0DY0UXDLetwLoRkIEHfe3FS8RcAA5F/ioLy3yPNHpSCKeUuln3ncTzlLvyHD/Pm3r0Ul7koKi6FHgG57tAaWAGor1ds3Cjwem06OixCIUFdHUSj6X51dWosERIi7Rb19WllTJ+WIwHxVJwzIsiDX1jMqa3/ZQAQDp0Kt4NkwS7uWWQQHiykPxbDTtq8sbmZr997L+U3zaR3Vx9qAEplESrb3PQi8PthyRI1ttPbtiIUOi9vvx8eeeSjz9MkZAJ9/Ac6JT1F/PPtP/OfQAfJEptZ5Tt5asWTNL6Z4HNf/hItf9vPO62tnIvFEBpMMis5tOMA/T19xGWMSal87Cy25tGDPEIDJGrkVCglBIPjj7vU81EbCO28/Y9g3E3QJw3WWmvo3z9MaZ+L2bKaLe1Pc3fjDNwfTubgW2+z5x+HaO84TV93D0aigrZfHafuL8uoOTeLyoSJL2Ggq7yMCXAARsomFrUYkBoer3lFgVQAHq/JgNSIRS2MlI3j430uVRKTSCwtfbrT0TFT6RTX72xlZcWPaU91opTClAavDbzI1JSPKXYpUa17rFxmpCrQGY+Ea18Sy70mOEIMMEZO9nXBT34ZGiuK3lHD1KtUFA1c7aLo1cPFP40poOVTKotflwTADf5hhJEJV35K77mhccMT8D8Jrm1SuSm/AAAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAy
```

Comment:
Note: we have a random module that you can import! https://github.com/tidbyt/pixlet/blob/main/docs/modules.md#pixlet-module-random